CONTRIBUTING.md

```diff
@@ -135,15 +135,15 @@ or
 
 * Install dependencies:
 
-``pip install -r requirements-dev.txt``
+``poetry install --extras "sqlserver"``
 
 * Go to the ``test_infra`` directory
 
 ``cd test_infra``
 
 * Install CDK dependencies:
 
-``pip install -r requirements.txt``
+``poetry install``
```

Comment:
Should it be `poetry install --no-dev` perhaps?

Reply:
Currently the `dev` dependencies section is empty so it should do exactly the same.